R/utility-commands.R

```diff
@@ -41,6 +41,13 @@ getPoolPackageInstallationCommand <- function(type, packages) {
             )
   }
   else if (type == "github") {
+    if (githubAuthenticationToken != "") {
+      installCommand <-
+        paste(installCommand,
+              sprintf("-e \'githubAuthToken <- \\\"%s\\\"\'", githubAuthenticationToken),
```

Comment:
do you need a space before -e?

Reply:
single quote ' strings to avoid needing to escape stuff?

Reply:
1) paste command adds a space automatically. paste0 would require me to have a space before -e
2) Single quote for escaping stuff.